astroid/modutils.py

```diff
@@ -51,18 +48,11 @@
 import sys
 import sysconfig
 import types
-from distutils.errors import DistutilsPlatformError  # pylint: disable=import-error
-from distutils.sysconfig import get_python_lib  # pylint: disable=import-error
 from pathlib import Path
 from typing import Dict, Set
 
 from astroid.interpreter._import import spec, util
 
-# distutils is replaced by virtualenv with a module that does
-# weird path manipulations in order to get to the
-# real distutils module.
-
-
 if sys.platform.startswith("win"):
```

Comment:
```suggestion

if sys.platform.startswith("win"):
```
Let's also keep the two empty lines here.

Reply:
Oh, well. Seems like black didn't like that 🤷🏻